docs/schema/actions-element-workflowinfo-elementaction4.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Actions element (WorkflowInfo element)
+description: Discusses how you can create custom workflow activities and expand the palette of workflow actions that you can use to model unique business processes.
```

Comment:
Let's add the API to the description.  Something like...

"Actions element discusses how you can create custom workflow activities and expand the palette of workflow actions that you can use to model unique business processes."

Reply:
That makes it too long!
How about: 
Discusses the use, elements and definitions for the Actions element (WorkflowInfo element) to expand the palette of workflow actions.

133 vs 168 :-D